task0/Python_Intro_Ferhat.ipynb

```diff
@@ -665,16 +665,35 @@
   },
```

Comment:
<pre class="ql-syntax" spellcheck="false">theta_1 = [
  [2, 1, 3],
  [1, 2, 3],
  [3, 1, 2]
]

x = [7, 8, 5]
theta_0 = [1, 1, 1]

# Output
y = [0, 0, 0]

# Since both all lists have
# the same number of elements.
for index, list in enumerate(theta_1):
  for value in list:
     y[index] += value * x[index]
  y[index] += theta_0[index]

print(y)

</pre><p> </p><p>I didn't try the code but probably this is enough for the operation.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/bogaziciai/assignment/pull/7/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='71'/>